build.gradle

```diff
@@ -29,3 +29,13 @@ dependencies {
 
 // Define the main class for the application
 mainClassName = 'App'
+
```

Comment:
the reason this is failing is because it's not up to date with master and thus doesn't have the shadowjar dependency added in.

You should either rebase with master, or merge master into your branch